README.md

```diff
@@ -42,83 +34,109 @@ Run `source environment`  now and whenever these environment files are modified.
 #### Configuring cloud-specific access credentials
 
 ##### AWS
-* Follow the instructions in http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html to get the
-  `aws` command line utility.
-* Create an S3 bucket that you want DSS to use.
-* In `environment.local`, set the environment variable `DSS_S3_BUCKET_TEST`.
-* If you wish to run the unit tests, you must create a second S3 bucket to store the test fixtures, and set the
-  environment variable `DSS_S3_BUCKET_TEST_FIXTURES` to the name of that bucket.
+
+* Follow the instructions in http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html to get the `aws` command line utility.
```

Comment:
There is inconsistent usage of bulleted lists vs numbered lists.  May not be relevant to your change, but FYI.

Reply:
I'll fix it. a555277507219b2d6357ae59bf2628e264045fab makes that a lot easier.